programs/staker/src/math.rs

```diff
@@ -16,10 +17,16 @@ pub fn calculate_reward(
     if current_time <= start_time {
         return Err(ErrorCode::NotStarted.into());
     }
+    let seconds_per_liquidity =
```

Comment:
idk if we want that implementation. It will be better to hide it behind abstraction like `FeeGrowth::unchecked_sub`

Reply:
done